test.py

```diff
@@ -349,6 +349,8 @@ def create_server(create_cfg: ScyllaCluster.CreateServerParams):
             cmdline_options = self.cfg.get("extra_scylla_cmdline_options", [])
             if type(cmdline_options) == str:
                 cmdline_options = [cmdline_options]
+            if create_cfg.cmdline_from_test is not None:
```

Comment:
You specified
```
        cmdline_from_test: List[str]
```
so it cannot be `None`, assuming we type-check the code.

I agree with the choice of `List[str]` instead of `Optional[List[str]]`, there's no point in passing `None`s when we can pass `[]`.